tap/source/discoverer_util.go

```diff
@@ -0,0 +1,38 @@
+package source
+
+import (
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"github.com/up9inc/mizu/shared/logger"
+)
+
+var numberRegex = regexp.MustCompile("[0-9]+")
+
+func readEnvironmentVariable(file string, name string) (string, error) {
```

Comment:
`getSingleValueFromEnvironmentVariableFile(filePath string, variableName string)` would be more clean